docs/swagger.yaml

```diff
@@ -931,6 +974,25 @@ definitions:
       userDetails:
         description: Vipps User Information
         "$ref": "#/definitions/UserDetails"
+  AuthorizationTokenResponse:
```

Comment:
response parameters have underscore in their titles. Actual result from a call returned following parameters:
token_type
expires_in
ext_expires_in
expires_on
not_before
resource
access_token

Reply:
Fixed and commited,